src/nifgen/metadata/nifgen.proto

```diff
@@ -0,0 +1,2150 @@
+
```

Comment:
These files got moved to a different dir.
Looks like this PR needs to merge latest main?

Reply:
No, only the shared .proto files got moved. (And that PR _just_ went in.) The per-driver files were left next to the .py metadata, for (hopefully) easier maintenance.

Reply:
🤔
I'm not sure which way is better. Alright this is good for this PR thanks for clarifying.